docs/contributing.md

```diff
@@ -190,3 +190,12 @@ To update a specific version with the latest changes:
 
 1. Remove the version from `versions.json`.
 1. Run `npm run docusaurus docs:version <version>`.
+
+### New Demo Page Added
```

Comment:
I'm confused 😆 😊  why do our contributors need this page?

I believe it is an extermely rare case when the contributor changes the global styling of the website without coordinating that with our team or the designers.

Also, we have plans to make automated checks on this page so if the contributors change something accidently, they'll get errors from CI.

Last but not least, the demo page has an explanation what it is for.

@asafkorem , I suggest rephrasing or removing this. WDYT?

Reply:
Yes, I agree with this section. I was just thinking about that when I glanced at the changes https://github.com/wix/Detox/pull/3829#discussion_r1067945205

Reply:
You're probably right about that ("rare case when the contributor changes the global styling of the website without coordinating that with our team or the designers"), but maybe it won't be an external contrib., but some future collaborator 😅 I generally think it makes sense to have this page linked from here but I don't have a strong opinion here

Reply:
Okay, @moutro, I'd suggest then rephrasing like "if you are working on the website stylesheets, check with this internal demo page to prevent undesirable visual regressions".

Reply:
Perhaps if I change the wording to be less like an announcement and more like a reference? 

Ex:
This Demo page serves as a visual representation of the styling and formatting of our documentation.
You can now access it by visiting the [Demo page](/demo) from the navigation menu.

We hope you'll find this new addition to be a valuable resource when contributing to the project.

Reply:
Sorry, @noomorph - your comment didn't arrive as I was working on the rephrasing. Should I change to your text?

Reply:
Thanks, I merged your version with mine. Could you please check it once again now?

Also, your `[Demo](/demo)` link was likely to be incorrect – you had to use either `[Demo](demo)` or `[Demo](demo.mdx)`, because that slash `/` would be generation urls like `/Detox/demo` while you actually want it to be `/Detox/docs/:version/demo`.